packages/dds/ink/src/ink.ts

```diff
@@ -213,12 +213,9 @@ export class Ink extends SharedObject<IInkEvents> implements IInk {
      * {@inheritDoc @fluidframework/shared-object-base#SharedObject.loadCore}
      */
     protected async loadCore(storage: IChannelStorageService): Promise<void> {
-        const blob = await storage.readBlob(snapshotFileName);
-        const header = bufferToString(blob, "utf8");
-        if (header !== undefined) {
```

Comment:
I'm looking at this code and I feel like this code was always wrong. I.e. was there any chance to have undefined header? readBlob() throws on missing blob, right? Even if it did not throw, bufferToString can't return undefined.
If it's all true, I'd rather stop checking for undefined in new code as well (or make it an assert)